cli/cmd/delete_cmd.go

```diff
@@ -1,21 +1,55 @@
 package cmd
 
 import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/actions"
+	"github.com/kubeshop/tracetest/cli/analytics"
+	"github.com/pterm/pterm"
 	"github.com/spf13/cobra"
+	"go.uber.org/zap"
 )
 
+var deletedResourceID string
+
 var deleteCmd = &cobra.Command{
 	Use:    "delete [resource type]",
 	Long:   "Delete resources from your Tracetest server",
 	Short:  "Delete resources",
 	PreRun: setupCommand(),
 	Args:   cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		cmd.Help()
+		resourceType := args[0]
+		ctx := context.Background()
+
+		analytics.Track("Resource Delete", "cmd", map[string]string{
+			resourceType: resourceType,
+		})
+
+		resourceActions, err := resourceRegistry.Get(actions.SupportedResources(resourceType))
+
+		if err != nil {
+			cliLogger.Error(fmt.Sprintf("failed to get resource instance for type: %s", resourceType), zap.Error(err))
+			os.Exit(1)
+			return
+		}
+
+		err = resourceActions.Delete(ctx, deletedResourceID)
+
+		if err != nil {
+			cliLogger.Error(fmt.Sprintf("failed to apply definition for type: %s", resourceType), zap.Error(err))
+			os.Exit(1)
+			return
+		}
+
+		cmd.Println(pterm.FgGreen.Sprintf(fmt.Sprintf("✔ Resource deleted successfully for resource type: %s", resourceType)))
 	},
 	PostRun: teardownCommand,
 }
 
 func init() {
+	deleteCmd.PersistentFlags().StringVarP(&deletedResourceID, "identifier", "i", "", "id of the resource to delete")
```

Comment:
are you sure this needs to be a persistent flag? If i'm not mistaken, persistent flags are "inherited" by subcommands, while "not-persistent" flags are specific to the command only.

Reply:
also, I think we defined the full flag to be `--id`, not `--identifier` (definition here: https://github.com/kubeshop/tracetest/issues/1937#issuecomment-1458493552). We probablly don't need a short version for the ID